operational/stranded_servers/README.md

```diff
@@ -0,0 +1,33 @@
+## Stranded Servers
+ 
+### What it does
+
+This policy checks all Servers in Cloud Management and reports on any that have stranded during the boot process. It provides the option to terminate any stranded servers after approval.
+
+### Functional Details
+ 
+The policy leverages Cloud Management APIs to identify servers that have not successfully completed their boot sequence (i.e. have "stranded in booting"). Servers can optionally be terminated after approval.
```

Comment:
It should probably be added that locked instances will not be terminated.

Reply:
I have updated.